src/node_http2.h

```diff
@@ -853,13 +854,16 @@ class Http2Session : public AsyncWrap, public StreamListener {
     current_session_memory_ -= amount;
   }
 
+  // Tell our custom memory allocator that this rcbuf is independent of
+  // this session now, and may outlive it.
+  void StopTrackingRcbuf(nghttp2_rcbuf* buf);
+
   // Returns the current session memory including the current size of both
   // the inflate and deflate hpack headers, the current outbound storage
```

Comment:
Might want to change the hpack headers part, as it is no longer visible in code?

Reply:
Yup, done!